cpu/atmega1281/Makefile.include

```diff
@@ -3,6 +3,9 @@ export CFLAGS += -DCOREIF_NG=1
 
 # tell the build system that the CPU depends on the atmega common files
 USEMODULE += atmega_common
+USEMODULE += pm_fallback_set_lowest
```

Comment:
can these be moved to atmega_common?

Reply:
@kaspar030 No, the idea is that the *final* CPU will *always* select everything it needs. In particular my implementation of the pm is specific to atmega1284p, as registers and modes and peripheral levels are individual per CPU.
The only thing would be to create a pm_fallback.mk.inc in atmega_common that will be included in Makefile.include of the CPU?